### PR TITLE
Ip 5938

### PR DESCRIPTION
**Pull request by thrineshactian**
added odbc init container

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -79,6 +79,15 @@ spec:
...
           mountPath: /opt/Actian/local/work/job
         {{- end }}
+      - name: init-dcodbc
+        image: {{ .Values.dcOdbcImage }}
+        imagePullPolicy: {{ .Values.dcOdbcImagePullPolicy }}
+        command: ['sh', 'script.sh']
+        securityContext:
+          {{- toYaml .Values.dcOdbcSecurityContext | nindent 10 }}
+        volumeMounts:
+        - name: datacloud-odbc-vol
```
Comment (jsnwgnr) [inline templates/deployment.yaml:89]: Please avoid abbreviation
/opt/Actian/datacloud-odbc

File: templates/deployment.yaml [hunk 2]
```diff
@@ -128,6 +137,9 @@ spec:
         - name: dataflow-engine-vol
           mountPath: /opt/Actian/dataflow
           readOnly: false
+        - name: datacloud-odbc-vol
```
Comment (jsnwgnr) [inline templates/deployment.yaml:140]: Please avoid abbreviation
/opt/Actian/datacloud-odbc

File: templates/deployment.yaml (hunk 1 above)
Comment (thrineshactian) [reply templates/deployment.yaml:89]: updated

File: templates/deployment.yaml (hunk 2 above)
Comment (thrineshactian) [reply templates/deployment.yaml:140]: updated

**Review by jsnwgnr — APPROVED** [review]

**Pull request merged**